tests/src/Behat/example.behat.local.yml

```diff
@@ -0,0 +1,31 @@
+# Local Behat configuration file.
```

Comment:
This is only useful if we really work on the Behat test suite. I would leave this out unless the testing plan changes.